scripts/novaseq/demux-novaseq.batch

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+#SBATCH -c 18
+#SBATCH --qos=low
```

Comment:
Should this not be high for Demux? Is there any process that exceeds a demux in urgency? Arguably there could be options. Per se, if it's on stage then we have low, if on prod it's high.